py-polars/polars/series/series.py

```diff
@@ -1836,6 +1860,12 @@ def nan_max(self) -> int | float | date | datetime | timedelta | str:
         This differs from numpy's `nanmax` as numpy defaults to propagating NaN values,
         whereas polars defaults to ignoring them.
 
+        Examples
+        --------
+        >>> s = pl.Series("a", [1, None, 4])
+        >>> s.nan_max()
```

Comment:
This should showcase a nan value, e.g. `float("nan")`.

Best to give two examples, one where the series has a NaN, one where it doesn't.